sros2/sros2/_utilities.py

```diff
@@ -24,29 +24,30 @@
 from cryptography.hazmat.primitives import serialization
 from cryptography.hazmat.primitives.asymmetric import ec
 
+import sros2.errors
+
 _DOMAIN_ID_ENV = 'ROS_DOMAIN_ID'
 _KEYSTORE_DIR_ENV = 'ROS_SECURITY_KEYSTORE'
 
 
-def create_symlink(*, src, dst):
-    if os.path.exists(dst):
-        src_abs_path = os.path.join(os.path.dirname(dst), src)
-        if os.path.samefile(src_abs_path, dst):
+def create_symlink(*, src: pathlib.Path, dst: pathlib.Path):
+    if dst.exists():
+        # Don't do more work than we need to
+        if dst.samefile(dst.parent.joinpath(src)):
```

Comment:
some functions use `joinpath()` and others ` / `. Is there something preventing us to use the same everywhere in sros2 ?

Reply:
You're right, nothing is preventing it, I just really dislike using `/` for this because its meaning changes depending on the operands, and thus decreases readability in my opinion. Do you prefer it?

Reply:
I like consistency, without strong feelings for one syntax over the other. If all SROS2 can use `joinpath` that sounds good to me :+1: